File: DClass.java
import java.lang.*;
import java.util.*;

class Check
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj=new Scanner(System.in);
		
		System.out.println("enter the number:");
		this.iNo=sobj.nextInt();
		if(iNo<0)
		{
			iNo=-iNo;
		}
	}
	public void DisplayClass()
	{
	if(iNo<35)
	{
		System.out.println("Fail");
	}
	else if((iNo>=35)&&(iNo<50))
	{
		System.out.println("first class");
	}
	else if((iNo>=50)&&(iNo<60))
	{
		System.out.println("second class");
	}
	else if((iNo>=60)&&(iNo<70))
	{
		System.out.println("first class");
	}
	else if(iNo>=70)
	{
		System.out.println("First class with distinction");
	}
	}
}
class DClass
{
	public static void main(String arg[])
	{
		Check cobj=new Check();
		cobj.Accept();
		cobj.DisplayClass();
	}
}

File: DClass.cpp
#include<iostream>
using namespace std;

class Check
{
	private:
	int iNo;
	
	public:
	void Accept()
	{
		cout<<"enter the number"<<"\n";
		cin>>this->iNo;
		if(iNo<0)
		{
			iNo=-iNo;
		}
	}
	void DisplayClass()
	{
	if(iNo<35)
	{
		cout<<"Fail"<<"\n";
	}
	else if((iNo>=35)&&(iNo<50))
	{
		cout<<"Pass class"<<"\n";
	}
	else if((iNo>=50)&&(iNo<60))
	{
		cout<<"second class"<<"\n";
	}
	else if((iNo>=60)&&(iNo<70))
	{
		cout<<"first class"<<"\n";
	}
	else if(iNo>=70)
	{
		cout<<"first class with distinction"<<"\n";
	}
	}
};
int main()
{
	Check cobj;
	cobj.Accept();
	cobj.DisplayClass();
	
	return 0;
}